libs/data/mlpe/data/transforms/preprocessor.py

```diff
@@ -49,9 +49,9 @@ def forward(self, strain: torch.Tensor, parameters: torch.Tensor):
         # we need to transpose the parameters tensor so that
         # we normalize along each individual attribute, and not each
         # individual sample
-        parameters.transpose(0, 1)
```

Comment:
Does any of the tests use this preprocessor? I found this one: https://github.com/ML4GW/PE/blob/main/libs/data/tests/test_dataloader.py#L88-L91 which is defining its own preprocessor. Can this test be changed to use this preprocessor class

Reply:
Also, in general, adding a unittest that shows what are the shapes of the inputs for whitener.fit or scaler.fit will be good

Reply:
Since the preprocessor is essentially a light wrapper of `ml4gw` modules that are already tested, I didnt think we needed any. I think as long as we explain via comments all the odd transposes we have to do we should be good